playbooks/roles/edxapp/tasks/deploy.yml

```diff
@@ -354,19 +373,19 @@
 - name: update supervisor configuration
   shell:  "{{ supervisor_ctl }} -c {{ supervisor_cfg }} update"
   register: supervisor_update
-  sudo_user: "{{ supervisor_service_user }}"
+  become_user: "{{ supervisor_service_user }}"
   changed_when: supervisor_update.stdout is defined and supervisor_update.stdout != ""
   when: not disable_edx_services
   tags:
     - manage
 
 - name: ensure edxapp has started
-  supervisorctl: >
-    state=started
-    supervisorctl_path={{ supervisor_ctl }}
-    config={{ supervisor_cfg }}
-    name="edxapp:{{ item }}"
-  sudo_user: "{{ supervisor_service_user }}"
+  supervisorctl:
+    name: "edxapp:"
+    supervisorctl_path: "{{ supervisor_ctl }}"
+    config: "{{ supervisor_cfg }}"
+    state: started
+  become_user: "{{ supervisor_service_user }}"
   when: celery_worker is not defined and not disable_edx_services
   with_items: service_variants_enabled
```

Comment:
@pomegranited I might be missing something, but in the new version of the code, there's just `name: "edxapp:"` (instead of `name: "edxapp:{{ item }}"`), but the new version still has that `with_items: service_variants_enabled` line. So how can the info that comes from `with_items` be taken into account by the `supervisorctl` module?

I think we'd want to run `supervisorctl` with `edxapp:lms`, `edxapp:cms`, etc. instead of running it with `edxapp:` multiple times for each item that `with_items` generates?